server/index.js

```diff
@@ -1,25 +1,33 @@
-/* eslint-disable import/no-extraneous-dependencies */
-const webpack = require('webpack');
-const webpackConfig = require('./config/webpack.config.dev');
-const webpackDevMiddleware = require('webpack-dev-middleware');
-const webpackHotMiddleware = require('webpack-hot-middleware');
 const express = require('express');
 const bodyParser = require('body-parser');
 const players = require('./api/players');
 const scorepads = require('./api/scorepads');
+const config = require('./config/config');
 
-const compiler = webpack(webpackConfig);
 const app = express();
 
+
+if (config.dev) {
```

Comment:
Hier wird das `config` Attribut verwenden, dass in der `config.js` eingetragen wird. Wenn `dev` eingeschaltet ist, werden die ganzen Dev-Features verwendet - auf dem Raspberry Pi wird dann `dev: false` gesetzt, und die Features alle disabled.